### PR TITLE
Fixes pre commit

### PR DESCRIPTION
**Pull request by paddyroddy**
Addresses quantumjot/btrack#275. Cannot test locally at the moment due to some `tox` issue, so will check on here. Some `isort` changes, see quantumjot/btrack#249.

### CODE REVIEW

**Review by p-j-smith — APPROVED** [review]
looks like linting is fixed but tests are still failing - let's fix the tests in a different pr

File: napari_btrack/track.py [hunk 1]
```diff
@@ -305,15 +305,13 @@ def _widgets_to_tracker_config(container: Container) -> TrackerConfig:
     for default_name, default_value in zip(
         ["measurements", "states", "dt", "prob_not_assign", "name"],
         [3, 6, 1.0, 0.001, f"{mode}_motion"],
-        strict=True,
```
Comment (paddyroddy) [inline napari_btrack/track.py]: Oh this is such an unsatisfying fix 😢

Comment (p-j-smith) [reply napari_btrack/track.py]: I think it's the only option unless we only support Python >= 3.10 😞

Comment (p-j-smith) [reply napari_btrack/track.py]: or we [implement the check ourselves](https://stackoverflow.com/a/69485272)... but that seems a little extreme - it's pretty easy to see here that each list has five elements

Comment (paddyroddy) [reply napari_btrack/track.py]: Yeah there's no point. I've ignored it with a comment in the config for now

**Pull request merged**